package.json

```diff
@@ -32,7 +33,7 @@
     "develop": "gatsby develop",
     "format": "prettier --write 'src/**/*.js'",
     "lint": "eslint 'src/**/*.js'",
-    "precommit": "yarn format && yarn lint --fix",
+    "precommit": "yarn format && yarn lint --fix && git add -A",
```

Comment:
this is to prevent having to commit again in the event that prettier makes changes during the precommit